docs/openstack/persistent-storage/format-and-mount-the-volume.md

```diff
@@ -0,0 +1,104 @@
+# Format And Mount The Volume
+
+## For Linux based virtual machine
+
+SSH to your instance. You should now see the volume as an additional disk in
+the output of `sudo fdisk -l` or `lsblk` or `cat /proc/partitions`.
+
+    # lsblk
+    NAME    MAJ:MIN RM  SIZE RO TYPE MOUNTPOINT
+    ...
+    vda     254:0    0   10G  0 disk
+    ├─vda1  254:1    0  9.9G  0 part /
+    ├─vda14 254:14   0    4M  0 part
+    └─vda15 254:15   0  106M  0 part /boot/efi
+    vdb     254:16   0    1G  0 disk
+
+We see the volume here as the disk 'vdb', which matches the `/dev/vdb/` we
+noted in "Attached To" column.
```

Comment:
Since the Attach to column was part of another page you might want to remind the user what you are referring to. or just end the sentence after 'vdb'